src/com/textFinder/finder/FileSearchThreadRunner.java

```diff
@@ -0,0 +1,28 @@
+package com.textFinder.finder;
+
+import java.io.File;
+import java.util.List;
+
+public class FileSearchThreadRunner implements Runnable {
+    private final File file;
+    private final String searchText;
+    private final List<File> result;
+
+    public FileSearchThreadRunner(File file, String searchText, List<File> result) {
```

Comment:
result arg and private field would both not be needed after the suggested changes